src/keycloak/admin/users.py

```diff
@@ -72,26 +89,115 @@ def by_id(self, user_id):
 
 
 class User(KeycloakAdminBase):
+    _BASE = "/auth/admin/realms/{realm}/users/{user_id}"
     _paths = {
-        'single': '/auth/admin/realms/{realm}/users/{user_id}'
+        'single': _BASE,
+        'group': _BASE + '/groups',
+        'groupAdd': _BASE + '/groups/{group_id}'
     }
 
     def __init__(self, realm_name, user_id, *args, **kwargs):
         self._realm_name = realm_name
         self._user_id = user_id
-
+        self._it = None
         super(User, self).__init__(*args, **kwargs)
 
+    @property
+    def it(self):
+        if self._it is None:
+            self.get()
+        return self._it
+
+    def roles(self):
```

Comment:
Can you rename this to "role_mappings"? Since that's how the API path actually is named.